torch_xla/core/dynamo_bridge.py

```diff
@@ -387,6 +446,12 @@ def optimized_mod(*args):
     nonlocal xla_args_need_update
     nonlocal skip_checking_input_sharding_threashold
 
+    original_device: torch.device = _get_input_arg_device(args)
+    is_cuda_args: bool = _args_on_cuda(args)
```

Comment:
`_args_on_cuda ` will call `_get_input_arg_device` which is redundant.

Reply:
I still think it's a little cleaner to do the redundant call, but removed the call here.